test/e2e/network/netpol/model.go

```diff
@@ -21,11 +21,28 @@ import (
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+const (
+	CPURequest    = "1m"
+	MemoryRequest = "50Mi"
```

Comment:
ran some experiments, imo this is a reasonable number but we could make it even smaller if we wanted to.  iirce k8s scheduler by default is in this region for binpacking  (100m)     
```
 docker container run -p 8000:8000 -d -t -i --memory-reservation=10m --name agn gcr.io/kubernetes-e2e-test-images/agnhost:2.6 net --serve 127.0.0.1:8000
```